lapixdl/formats/annotation.py

```diff
@@ -152,6 +154,49 @@ def to_polygon(self) -> Polygon:
         return Polygon(self.coords)
 
 
+@dataclass
+class Annotation:
+    geometry: Polygon | MultiPolygon
+    category_id: int
+    iscrowd: int = 0
+
+    @property
+    def bbox(self) -> BBox:
+        return bounds_to_BBox(self.geometry.bounds, self.category_id)
+
+    @property
+    def _geo_type(self) -> str:
+        return self.geometry.geom_type
+
+    @property
+    def coco_bbox(self) -> list[float | int]:
```

Comment:
```suggestion
    def xywh_bbox(self) -> list[float | int]:
```
In wich cases the coordinates will have float values? Aren't they pixel positions?

Reply:
I don't know, I get this from https://cocodataset.org/#format-data at some months ago, but seems to have nothing about now. For me, we can use just as int too.